mitiq/benchmarks/rotated_randomized_benchmarking.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) Unitary Fund
+#
+# This source code is licensed under the GPL license (v3) found in the
+# LICENSE file in the root directory of this source tree.
+
+"""Functions for generating rotated randomized benchmarking circuits."""
+from typing import List, Optional, cast
+
+import cirq
+import numpy as np
+
+from mitiq import QPROGRAM
+from mitiq.benchmarks import generate_rb_circuits
+from mitiq.interface import convert_from_mitiq
+
+
+def generate_rotated_rb_circuits(
+    n_qubits: int,
+    num_cliffords: int,
+    theta: Optional[float] = None,
+    trials: int = 1,
+    return_type: Optional[str] = None,
+    seed: Optional[int] = None,
+) -> List[QPROGRAM]:
+    r"""
+    Generates a list of "rotated" randomized benchmarking circuits.
+    This benchmarking method enables testing QEM techniques in more general
+    scenarios, closer to real-world applications in which expectation values
+    can take arbitrary values.
+
+    Rotated randomized bencmarking circuits are randomized benchmarking
+    circuits with an :math:`R_z(\theta)` rotation inserted in the middle, such
+    that:
+
+    .. math::
+        C(\theta) =    G_n \dots G_{n/2 +1} R_z(\theta)G_{n/2} \dots G_2 G_1
+
+    where :math:`G_j` are Clifford elements or Clifford gates.
+
+    The circuits generate expectation values which are sinusoidal functions of
+    :math:`\theta`, which in the ideal (noiseless) case vary in a continuous
+    interval of :math:`E_{\rm ideal} \in  [-1, 1]`.
```

Comment:
To avoid unexpected results, I would say something similar to this:

```suggestion
    For most values of the seed, the probability of the zero state is a
    sinusoidal function of :math:`\theta`. For some values of the seed
    the probability of the zero state is 1 for all :math:`\theta`.
```